runtime/oti/j9nonbuilder.h

```diff
@@ -4291,6 +4291,10 @@ typedef struct J9JITConfig {
 	uint64_t serverUID;
 #endif /* J9VM_OPT_JITSERVER */
 	void (*jitAddNewLowToHighRSSRegion)(const char *name, uint8_t *start, uint32_t size, size_t pageSize);
+#if defined(J9VM_OPT_JITSERVER)
+	void *serverAOTMethodSet;
+	UDATA serverAOTQueryThread;
+#endif // J9VM_OPT_JITSERVER
```

Comment:
This code is included in C source files where `//` does not legally begin a comment.
This line should instead be:
```c
#endif /* defined(J9VM_OPT_JITSERVER) */
```